(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'NoneType' object has no attribute 'posts_author'"
p3
sS'layer'
p4
S'/mnt/c/Users/Ziming/Desktop/UCSC-Fall-18/CMPS183/said-it/applications/init/controllers/api.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f2a883a93b0, file "/mnt/c/Users/Ziming/Desktop/UCSC-Fall-18/CMPS183/said-it/applications/init/controllers/api.py", line 1>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/mnt/c/Users/Ziming/Desktop/UCSC-Fall-18/CMPS183/said-it/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/mnt/c/Users/Ziming/Desktop/UCSC-Fall-18/CMPS183/said-it/applications/init/controllers/api.py", line 44, in <module>\n  File "/mnt/c/Users/Ziming/Desktop/UCSC-Fall-18/CMPS183/said-it/gluon/globals.py", line 421, in <lambda>\n    self._caller = lambda f: f()\n  File "/mnt/c/Users/Ziming/Desktop/UCSC-Fall-18/CMPS183/said-it/gluon/tools.py", line 3867, in f\n    return action(*a, **b)\n  File "/mnt/c/Users/Ziming/Desktop/UCSC-Fall-18/CMPS183/said-it/applications/init/controllers/api.py", line 33, in edit_posts\n    if post.posts_author != auth.user.email:\nAttributeError: \'NoneType\' object has no attribute \'posts_author\'\n'
p11
s.